### DISCUSSION TITLE
ESP32S2 boards

### DISCUSSION THREAD

**Discussion by fedetarrats1** [lathoub/Arduino-BLE-MIDI#54] [Ideas]
Hello, I love the work you've done here, just wondering would it be too much to ask for compatibility with ESP32-S2 boards? 
Thank you

**Comment by fedetarrats1** [comment]
Nevermind, just realized S2 doesn't have BLE

**Reply by jhsa** [reply]
Hi, try the new S3 boards ;) I have just got one and it seems to be working. Check the post I have just made a few minutes ago.
